src/backend/BackendInterface.h

```diff
@@ -10,6 +10,10 @@
 
 namespace Backend {
 
+/**
```

Comment:
All these doxy comments should probably not be part of this pr either. Basically all changes but the limit line should not be in this pr.